src/ai/backend/manager/models/scaling_group.py

```diff
@@ -315,6 +320,45 @@ class ScalingGroup(graphene.ObjectType):
     scheduler_opts = graphene.JSONString()
     use_host_network = graphene.Boolean()
 
+    # Dynamic fields.
+    agent_count = graphene.Int(description="Added in 24.03.0.")
+    agent_total_resource_slots = graphene.JSONString(description="Added in 24.03.0.")
```

Comment:
The version those fields will be added should be `24.03.5` or later than that. Please check the latest release version and match the version notations.